### ISSUE TITLE
Any_value aggregate function wrong results.

### ISSUE THREAD

**Issue by laithsakka** [facebookincubator/velox#7745]
### Description

```
Expected 952, got 952
7 extra rows, 7 missing rows
7 of extra rows:
	3235985779296984255 | null | [13423,18857,8842] | null | 17714
	3235985779296984255 | [] | [13423,18857,8842] | null | 4880
	4969723216508924210 | null | null | [{"key":"9367919-08-27T23:11:20.114000000","value":65},{"key":"34605607-10-08T05:30:04.583000000","value":74},{"key":"63193939-09-06T10:50:31.186000000","value":23},{"key":"237251608-09-25T01:40:30.065000000","value":107}] | 30252
	4969723216508924210 | null | [19610,null,240,null,11399,29340,9568,30852,null,9494,null,24626,9008,11092,8663,31512] | null | 15430
	4969723216508924210 | null | [19610,null,240,null,11399,29340,9568,30852,null,9494,null,24626,9008,11092,8663,31512] | [{"key":"49354957-05-18T02:36:42.899000000","value":115},{"key":"92527710-11-15T23:26:58.461000000","value":37},{"key":"133088130-10-22T18:39:58.656000000","value":16},{"key":"232265860-01-04T03:18:16.433000000","value":63},{"key":"245280705-05-26T10:43:19.271000000","value":null},{"key":"283274942-09-11T08:01:59.652000000","value":35},{"key":"292096464-10-18T22:58:06.974000000","value":12}] | 19066
	4969723216508924210 | [{"key":"JFZYJzF4bF9zIm18MHt2W1U5bHJlMjlKbm8w
	```

### Error Reproduction

Meta only : 
https://www.internalfb.com/intern/test/844425007734525?ref_report_id=0

Non-Meta: 
https://github.com/facebookincubator/velox/actions/runs/6993064674/job/19025351914

**Comment by amitkdutta** [comment]
Thanks @laithsakka. https://github.com/facebookincubator/velox/pull/7748 should fix this issue.

**Comment by mbasmanova** [comment]
Fixed in #7748
